cob_bringup/robots/raw3-1-calib.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0"?>
```

Comment:
what's the difference between ```raw3-1``` and ```raw3-1-calib```?
there should only be one setup for each physically existing robot